<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>2400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="device2_light_icon_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>1110</y>
      <width>120</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/rec/img/light_off.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fanSpeed_text_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>1740</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>风速</string>
    </property>
   </widget>
   <widget class="QLabel" name="device2_light_intensity_text_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>1190</y>
      <width>401</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>灯光控制</string>
    </property>
   </widget>
   <widget class="QLabel" name="gas_text_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>870</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>气体</string>
    </property>
   </widget>
   <widget class="QLabel" name="light_intensity_icon_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/img/light.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="temperture_icon_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/img/temperture.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Soil">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>760</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="light_checkBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>1130</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:unchecked {
image: url(:/rec/img/close.png);
}
QCheckBox::indicator:checked {
image: url(:/rec/img/open.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="soil_icon_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>790</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/img/soil.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="device1_light_intensity_text_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>灯光状态</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Light">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_FanSpeed">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1530</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Humidity">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConnectServer">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>2000</y>
      <width>791</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>未连接到服务器</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Temperture">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Air">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>760</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="humidity__text_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLabel" name="light_intensity_text_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>光强</string>
    </property>
   </widget>
   <widget class="QLabel" name="soil_text_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>870</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>土壤</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperture_text_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(115, 210, 22);*/</string>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidity_icon_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/img/humidity.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectServer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>1990</y>
      <width>471</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_FanSpeed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>1470</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gas_icon_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>790</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/img/gas.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="device1_light_icon_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>120</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/rec/img/light_off.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
